EarlySamples/InitialProject/InitialProject/Assets/Scripts/Test2/PlayerMovementHandler.cs

```diff
@@ -0,0 +1,79 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+public class PlayerMovementHandler : MonoBehaviour
+{
+    Vector3 movementVector;
+    Quaternion rotation;
+    [SerializeField] private float impulseForce;
+    [SerializeField] Rigidbody bodyToMove;
+    [SerializeField] Transform bodyToRotate;
+    bool moveImplemented;
+
+    private void Awake()
+    {
+        movementVector = Vector3.zero;
+        rotation = Quaternion.Euler(0, 0, 0);
+        moveImplemented = true;
+        impulseForce = FixedConstants.impulseForce;
+    }
+
+    private void OnEnable()
+    {
+        TestPlayerInputManager.moveDown += TestPlayerInputManagerMoveDown;
+        TestPlayerInputManager.moveUp += TestPlayerInputManagerMoveUp;
+        TestPlayerInputManager.moveRight += TestPlayerInputManagerMoveRight;
+        TestPlayerInputManager.moveLeft += TestPlayerInputManagerMoveLeft;
+        TestPlayerInputManager.jump += TestPlayerInputManagerJump;
+    }
+
+    private void TestPlayerInputManagerJump(object sender, System.EventArgs e)
```

Comment:
Names are not very good. What if TestPlayerInputManager was extracted to an interface? What would you call them then?

Reply:
Names changed to AddBackwardsForce, AddRightwardsForce etc.